Library/Formula/rakudo-star.rb

```diff
@@ -28,15 +21,35 @@ def install
     ENV.prepend "CPPFLAGS", "-I#{libffi.lib}/libffi-#{libffi.version}/include"
 
     ENV.j1  # An intermittent race condition causes random build failures.
+
+    backends = ["moar"]
+    generate = ["--gen-moar"]
+
     if build.with? "jvm"
-      system "perl", "Configure.pl", "--prefix=#{prefix}", "--backends=parrot,jvm", "--gen-parrot"
-    else
-      system "perl", "Configure.pl", "--prefix=#{prefix}", "--backends=parrot", "--gen-parrot"
+      backends << "jvm"
     end
+    if build.with? "parrot"
+      backends << "parrot"
+      generate << "--gen-parrot"
+    end
+    system "perl", "Configure.pl", "--prefix=#{prefix}", "--backends=" + backends.join(","), *generate
     system "make"
     system "make install"
-    # move the man pages out of the top level into share.
-    mv "#{prefix}/man", share
+
+    # TEMPORARY Workaround for http://stackoverflow.com/questions/9988125/shebang-pointing-to-script-also-having-shebang-is-effectively-ignored.
+    # rakudo-star 2014.12.1 has shebang lines in some scripts which point to other scripts with shebang
+    # lines, which will be ignored/fail on MacOS. This uses /usr/bin/env instead which is also implemented
+    # upstream (not released yet).
```

Comment:
Unfortunately this is the latest upstream release of rakudo star. The
one that is now in homebrew currently is outdated and I'd like to fix
that. I'll also provide an update as soon as there is a new version.

Am 03.02.15 um 10:15 schrieb Mike McQuaid:
> In Library/Formula/rakudo-star.rb
> <https://github.com/Homebrew/homebrew/pull/36456#discussion_r23990906>:
> 
>>      system "make"
>>      system "make install"
>> -    # move the man pages out of the top level into share.
>> -    mv "#{prefix}/man", share
>> +
>> +    # TEMPORARY Workaround for http://stackoverflow.com/questions/9988125/shebang-pointing-to-script-also-having-shebang-is-effectively-ignored.
>> +    # rakudo-star 2014.12.1 has shebang lines in some scripts which point to other scripts with shebang
>> +    # lines, which will be ignored/fail on MacOS. This uses /usr/bin/env instead which is also implemented
>> +    # upstream (not released yet).
> 
> I'd be tempted to instead update to the latest upstream release that
> doesn't have this problem or wait for the fixed upstream releases.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/36456/files#r23990906>.
> 

-- 
Martin Gruner
Senior Developer R&D

OTRS AG
Bahnhofplatz 1a
94315 Straubing

T: +49 (0)6172 681988 0
F: +49 (0)9421 56818 18
I:  www.otrs.com/

Geschäftssitz: Bad Homburg, Amtsgericht: Bad Homburg, HRB 10751,
USt-Nr.: DE256610065
Aufsichtsratsvorsitzender: Burchard Steinbild, Vorstand: André
Mindermann (Vorsitzender), Christopher Kuhn, Sabine Riedel

Schlanker, schneller und flacher denn je - OTRS 4! Und für alle, die
MEHR wollen: Entdecken Sie hier die OTRS Business Solution™ mit mehr
Business Features!
https://www.otrs.com/otrs-business-solution-fuer-besseren-kundenservice/?lang=de